notebooks/vertex_genai/solutions/semantic_search_with_vector_search_and_palm.ipynb

```diff
@@ -164,7 +164,7 @@
     "vectors = []\n",
     "\n",
     "for i in range(0, len(metadata), MAX_BATCH_SIZE):\n",
-    "    batch = metadata.abstract[i : i + MAX_BATCH_SIZE]\n",
+    "    batch = metadata.abstract[i : i + MAX_BATCH_SIZE].to_list()\n",
```

Comment:
@BenoitDherin Added this line due to the additional type change introduced in 1.36.1. (Now it doesn't take pd.Series.)

https://github.com/googleapis/python-aiplatform/commit/5294972b284596966bbf722102c59c8ca48d4515